Makefile

```diff
@@ -1,15 +1,27 @@
+CC=gcc
```

Comment:
Why are you setting CC here? If I want to run Clang's static-build, it tries to set CC itself. This will unset it right?